### PR TITLE
po: Add Georgian translation

### PR DESCRIPTION
**Pull request by NorwayFun**

### CODE REVIEW

File: po/ka.po [hunk 1]
```diff
@@ -0,0 +1,603 @@
...
+
+#: src/exp-html.c:127
+msgid "Include HTML page header"
+msgstr "HTML გვერდის თავსართის ჩასმა"
+
+#. TRANSLATORS: "lang=\"en\" refers to the page title
+#. "Teletext Page ...". Please specify "de", "fr", "es" etc.
+#: src/exp-html.c:256
+msgid "<title lang=\"en\">"
+msgstr "<title lang=\"en\">"
```
Comment (ileanadumitrescu95) [inline po/ka.po:125]: Should this line contain "ka\" instead of "en\" ?

**Comment by NorwayFun** [comment]
Hi.
Thought it's just an example, not a list of actual languages. If it is so,
let me know  and I'll correct it.
Temuri

**Comment by ileanadumitrescu95** [comment]
It would be consistent with the other .po files if that line 125 were changed to "ka". So I would prefer you make the correction. Thank you!

**Comment by NorwayFun** [comment]
No problem. Patch is updated.

**Pull request merged**